src/ScaleIdentification.jl

```diff
@@ -0,0 +1,107 @@
+using MIDI
+
+
+"""
+    MIDI_to_notes
+
+Returns the note names of an array of MIDI pitch value, inside of single octave.
+
+Example :
+julia> Midi_to_notes([0,13,26])
+["C","C","C"]
+
+"""
+function MIDI_to_notes(MIDInotes::Array{})
```

Comment:
The entirety of the function `MIDI_to_notes` does not have a reason to exist. MIDI.jl already provides `pitch_to_name` function. 

Therefore your function should simply be:
```julia
MIDI_to_notes(notes) = [pitch_to_name(n.pitch)[1:end-1] for n in notes]
```

Reply:
```
pitch_to_fundamental(notes) = [MIDI.PITCH_TO_NAME[mod(n.pitch, 12)] for n in notes]
```
This solution is obviously the best, but the attribute ```PITCH_TO_NAME``` is referenced nowhere in the ```doc``` so how I am suppose to know it exists ?